workspace/src/main/kotlin/com/cosmotech/workspace/service/WorkspaceServiceImpl.kt

```diff
@@ -335,7 +343,10 @@ internal class WorkspaceServiceImpl(
       val pageable: Pageable =
           Pageable.ofSize(csmPlatformProperties.twincache.workspace.defaultPageSize)
       val workspaces = workspaceRepository.findByOrganizationId(organizationId, pageable).toList()
-      workspaceRepository.deleteAll(workspaces)
+      workspaces.forEach {
```

Comment:
Why not make use of `WorkspaceService.deleteWorkspace` that cleanly delete the workspace and its data ?

Reply:
Because `WorkspaceService.deleteWorkspace` checks RBAC rights.
As we respond to a OrganizationUnredistered event, RBAC should be bypassed.